tools/api-markdown-documenter/examples/ExtendDocumentationDomain.ts

```diff
@@ -33,6 +37,28 @@ const sectionNode: SectionNode = new SectionNode(
 	new HeadingNode("Section with custom children!"),
 );
 
+const document = new DocumentNode({
```

Comment:
Note for reviewers: expand example to include extending `documentToMarkdown` as well